src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -138,6 +151,6 @@ public void clear() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof UniquePersonList // instanceof handles nulls
-                        && this.internalList.equals(((UniquePersonList) other).internalList));
+                && this.internalList.equals(((UniquePersonList) other).internalList));
```

Comment:
This change violates the coding standard (are `||` and `&&` at the same level here?)

Reply:
Caused by auto formatting as well. Will look into my Java code style settings in awhile.